charts/fluent-bit/values.yaml

```diff
@@ -54,6 +59,10 @@ openShift:
     create: true
     name: ""
     annotations: {}
+    runAsUser:
+      type: RunAsAny
+    seLinuxContext:
+      type: MustRunAs
```

Comment:
I left the default to `MustRunAs` since most of the users will not need to change the selinux context.